irs/src/index.js

```diff
@@ -1,19 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App'; 
+import App from './App';
 import rootReducer from './reducers';
 import { createStore, applyMiddleware } from 'redux';
 import { Provider } from 'react-redux';
-import thunk from 'react-thunk';
+import thunk from 'redux-thunk';
```

Comment:
good catch, I missed this the first time. make sure Erick adjusts this on his machine as well if he hasn't already